Description: Remove googleanalytics from sphinx configuration
 The sphinx extension googleanalytics is not available from the Debian archive
 and is not mandatory for building the documentation thus this patch remove it
 from the configuration.
Author: Michael Fladischer <FladischerMichael@fladi.at>
Last-Update: 2012-01-14
Forwarded: not-needed

--- a/docs/conf.py.in
+++ b/docs/conf.py.in
@@ -27,7 +27,7 @@
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = []
 
-extensions = ['sphinxcontrib.googleanalytics']
+#extensions = ['sphinxcontrib.googleanalytics']
 
 # Google 
 googleanalytics_id = 'UA-23928702-1'
